plugins/speculation-rules/hooks.php

```diff
@@ -12,48 +12,18 @@
 }
 // @codeCoverageIgnoreEnd
 
-/**
- * Prints the speculation rules.
- *
- * For browsers that do not support speculation rules yet, the `script[type="speculationrules"]` tag will be ignored.
- *
- * @since 1.0.0
- */
-function plsr_print_speculation_rules(): void {
```

Comment:
Note for reviewers: This function was moved without _any_ changes to the `plugin-api.php` file.